dev/config/keycloak/import/ltt-realm.json

```diff
@@ -156,7 +156,8 @@
          "backchannel.logout.session.required": "true",
          "backchannel.logout.revoke.offline.tokens": "false",
          "frontchannel.logout.url": "${KEYCLOAK_SHL_CREATOR_BASE}/logout",
-         "backchannel.logout.url": ""
+         "backchannel.logout.url": "",
+         "post.logout.redirect.uris": "${KEYCLOAK_CPRO_LANDING_PAGE}"
```

Comment:
@daniellrgn 

I can't find any matching env var elsewhere, where is this configured?